barcalendar.py

```diff
@@ -19,6 +19,10 @@
 import colorsys
 import datetime
 import itertools
+import re
+import requests
+import time
+import yaml
 
```

Comment:
```suggestion
import time

import requests
import yaml
```
Sorry, this was my fault: third-party imports should be in a new import section.